subprojects/core/src/main/java/org/gradle/internal/operations/trace/CompleteBuildOperation.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.operations.trace;
+
+import java.util.Map;
+
+public class CompleteBuildOperation {
```

Comment:
Can we rename this to something else than `xxxBuildOperation`? Only the things that we actually execute should be called `xxxBuildOperation`.